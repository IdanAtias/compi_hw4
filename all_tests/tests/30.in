K x = 1 K;
bool b = x > 1 K;
while ( true and b){
	K z = 10 K;
	while( false or b){
		K y;
		input y;
		if ( y >= x ){
			print "impossible" ;		
		}
	}
}
C y = 100 C;
%num_______ = 10; 
