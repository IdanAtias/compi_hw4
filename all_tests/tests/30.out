-- end scope --
-- end scope --
var:y:3:KELVIN
-- end scope --
var:z:2:KELVIN
line 14: lexical error
